extensions/netty/deployment/src/main/java/io/quarkus/netty/deployment/NettyProcessor.java

```diff
@@ -83,7 +83,7 @@ NativeImageConfigBuildItem build(BuildProducer<ReflectiveClassBuildItem> reflect
                 //.addNativeImageSystemProperty("io.netty.noUnsafe", "true")
                 // Use small chunks to avoid a lot of wasted space. Default is 16mb * arenas (derived from core count)
                 // Since buffers are cached to threads, the malloc overhead is temporary anyway
-                .addNativeImageSystemProperty("io.netty.allocator.maxOrder", "1")
+                .addNativeImageSystemProperty("io.netty.allocator.maxOrder", "3")
```

Comment:
If it's just about gRPC, shouldn't we have a build item that allows to increment that and we would collect them and add what's needed here?

Because AFAICS we reduced it to avoid some wasted space?

Reply:
Well either that or a build item pushing max values and we take the max of all of them.

Reply:
Yes we could do that. @michalszynkiewicz Wdyt?

Reply:
PR for it: https://github.com/quarkusio/quarkus/pull/15914